<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>face_reco</class>
 <widget class="QWidget" name="face_reco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="GroupBox_cam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>591</height>
    </rect>
   </property>
   <property name="title">
    <string>摄像头实时画面</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_show_camera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400; color:#ff0000;&quot;&gt;摄像头未开启&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout_cam">
     <item>
      <widget class="QCheckBox" name="CheckBox_uselocalcamera">
       <property name="text">
        <string>使用本地摄像头</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_opencam">
       <property name="text">
        <string>打开摄像头</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout_collect_facedata">
     <item>
      <widget class="QLabel" name="label_tip">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>基于dlib及opencv的视频人脸识别</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBox_facereco">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>人脸识别</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="GroupBox_Debug">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Debug设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="CheckBox_debug">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>调试模式</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="dbUserCountHorizontalLayout_2">
      <item>
       <widget class="QLabel" name="Label_confidenceThreshold">
        <property name="text">
         <string>置信度阈值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="SpinBox_Threshold">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.400000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="dbUserCountHorizontalLayout">
     <item>
      <widget class="QLabel" name="Label_facescnts">
       <property name="text">
        <string>数据库已存人脸样本数：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="LcdNum_faces">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="GroupBox_tips">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>使用说明</string>
    </property>
    <widget class="QLabel" name="confidenceTipLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>① 置信度阈值越小，精度要求越高！</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="debugLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PS：以上调整只有开启调试模式才会触发</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBox_log">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>290</y>
     <width>341</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>系统日志打印</string>
   </property>
   <widget class="QTextEdit" name="TextEdit_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
